glass/src/app/native/cpp/main.cpp

```diff
@@ -181,7 +183,29 @@ int main() {
       gPlotProvider->DisplayMenu();
       ImGui::EndMenu();
     }
+
+    bool about = false;
+    if (ImGui::BeginMenu("Info")) {
+      if (ImGui::MenuItem("About")) {
+        about = true;
+      }
+      ImGui::EndMenu();
+    }
     ImGui::EndMainMenuBar();
+
+    if (about) {
+      ImGui::OpenPopup("About");
+      about = false;
```

Comment:
Do you need the "about" variable here?  You can't just call OpenPopup from the if (MenuItem())?

Reply:
Oh... I guess its ID wouldn't be right, because it'd be pushed two layers deep?

Reply:
There is a problem with opening popups in menus. See: https://github.com/ocornut/imgui/issues/331

Reply:
Yeah makes sense given the ID stack.